File: temas/encapsulacion/javascript/Libro.java
import java.util.*;

 public class Libro 
 {
  private String titulo;
  private String autor;
  private int numPaginas;
  public Libro(String titulo, String autor, int numPaginas) {
    this.titulo = titulo; // variable privada para el título del libro
    this.autor = autor; // variable privada para el autor del libro
    this.numPaginas = numPaginas; // variable privada para el número de páginas del libro
  }
    // métodos públicos para acceder a las variables privadas
    public String getTitulo() {
      return titulo;
    }
    
    public String getAutor() {
      return autor;
    }
    
    public int getNumPaginas() {
      return numPaginas;
    }
  
  // método público para imprimir la información del libro
  public void imprimirInfo() {
    System.out.println("El titulo es: " + getTitulo() + ".\n");
    System.out.println("El autor es: " + getAutor()+".\n");
    System.out.println("El numero de paginas es: " + getNumPaginas() + ".\n");
  }

}

File: temas/encapsulacion/Python/Libro.py
class Libro:
    def __init__(self, titulo, autor, precio):
        self.__titulo = titulo
        self.__autor = autor
        self.__precio = precio

    def get_titulo(self):
        return self.__titulo

    def get_autor(self):
        return self.__autor

    def get_precio(self):
        return self.__precio

    def set_precio(self, precio):
        self.__precio = precio
